test-app/tests/integration/components/form/file-input/delete-button-test.gts

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable no-undef -- Until https://github.com/ember-cli/eslint-plugin-ember/issues/1747 is resolved... */
+/* eslint-disable simple-import-sort/imports,padding-line-between-statements,decorator-position/decorator-position -- Can't fix these manually, without --fix working in .gts */
+import {
+  render,
+  triggerEvent,
+} from '@ember/test-helpers'
+import { module, test } from 'qunit';
+
+import DeleteButton from '@crowdstrike/ember-toucan-core/components/form/file-input/delete-button';
+import { setupRenderingTest } from 'test-app/tests/helpers';
+
+// note: linter complains about importing this from another test
+type Options = {
+  name: string;
+  type?: string;
+};
+
+// https://medium.com/@chrisdmasters/acceptance-testing-file-uploads-in-ember-2-5-1c9c8dbe5368
+export function createFile(
+  content = ['Some sample content'],
+  options: Options = { name: '', type: '' }
+) {
+  const { name, type } = options;
+
+  const file = new File(content, name, {
+    type: type ? type : 'text/plain',
+  });
+
+  return file;
+}
+
+module('Integration | Component | Form | FileInput | DeleteButton', function (hooks) {
+  setupRenderingTest(hooks);
+
+
```

Comment:
Just wondering here: @nicolechung do you have prettier working for locally with gts files? It seems it _should_ have removed those double blank lines usually!

Also wondering, if that is not correctly formatted, then why is this not failing in CI? We do run `format:check:` there. Maybe something wrong with our prettier setup? 

Nothing to fix this PR IMO, but maybe we should look into this later.

Reply:
Been running `pnpm format` over the packages manually, with the exception of the `markdown` files which I format in VS Code. I might have missed this, will run it again.

Reply:
Uh, you are right. Tried that in headless, it's the same there (the command is different, like `pnpm lint:prettier:fix`), prettier doesn't even run on .gts files. It's only VS Code that is doing this for me ootb...